wordpress-mobile/WordPress-Android
# Replace Volley with Glide - ReaderBlogFragment

Fixes #8086 

Replaces Volley with Glide in the Reader - Site you may like/Followed Sites sections. Nothing should change from the user perspective.

To test:
1. Go to Reader
2. Click on the gear icon
3. Select tab `Followed sites`
4. Make sure all the images are correctly loaded 
5. Repeat steps 4 and 5 for tab `Site you may like`

Note: Some sites have site icon in svg format, which is not currently supported. I've created an enhancement #8073. Make sure all the images which are loaded in the Google Play version of the app are correctly loaded after this change.
